### DISCUSSION TITLE
[preview-tui] Stop previewing when performing action.

### DISCUSSION THREAD

**Discussion by duffydack** [jarun/nnn#1809] [Ideas]
I like the new video previews, much better than just a (sometimes crap) thumbnail, but, when I open the file I'm hovering to play it, the preview is still happily previewing away.  Can something be adapted to kill this preview after opening it?

**Comment by luukvbaal** [comment]
Yes, the restorepreview patch does exactly that.

**Reply by duffydack** [reply]
Sorry, I'm building nnn-git with that patch set but it's not doing what I expected - stop preview when I open it.  I must be missing something.

**Reply by luukvbaal** [reply]
Not sure sorry, it's been a while since I used it and can't test it currently. But yes that's what it is supposed to do.

**Reply by luukvbaal** [reply]
Sorry for misleading, the restorepreview patch only works when opening files in the attached editor. See #1814, where changing that is being considered.

**Reply by duffydack** [reply]
Just an update.  Since I'll never need to see it for more than 5 seconds, I just added a `timeout 5` to the mpv command. :)
